tests/test_challenges.py

```diff
@@ -30,26 +31,26 @@ def setup(self):
         responses.add(responses.GET, url.format(API_HOST_URL, URLS.future_challenge_list.value),
                       json=challenge_data, status=200)
 
-        challenges = challenge_data["results"]
+        challenges_json = challenge_data["results"]
```

Comment:
@isht3 What is the reason for renaming the variable?

Reply:
It was shadowing the imported variable.

Reply:
ohh...okay!